pocs/camera/libasi.py

```diff
@@ -97,6 +131,18 @@ def get_camera_property(self, camera_index):
                                                                pythonic_info['name']))
         return pythonic_info
 
+    def get_camera_property_by_id(self, camera_ID):
+        """Get properties of the camera with a given integer ID."""
+        camera_info = CameraInfo()
+        self._call_function('ASIGetCameraPropertyByID',
+                            camera_ID,
+                            ctypes.byref(camera_info))
+
+        pythonic_info = self._parse_info(camera_info)
+        self.logger.debug("Got info from camera {}, {}".format(pythonic_info['camera_ID'],
```

Comment:
Just a note that you could do `"{camera_ID}, {name}".format(**pythonic_info)` if you wanted. Might keep it on one line. ;)

And where are those multi-line suggestions when you need them!